Utilities/DataFlow/src/runSubframeBuilderDevice.cxx

```diff
@@ -13,15 +13,21 @@
 
 namespace bpo = boost::program_options;
 
+constexpr uint32_t o2::DataFlow::SubframeBuilderDevice::DefaultOrbitDuration;
+constexpr uint32_t o2::DataFlow::SubframeBuilderDevice::DefaultOrbitsPerTimeframe;
+
 void addCustomOptions(bpo::options_description& options)
 {
   options.add_options()
-//    (AliceO2::DataFlow::SubframeBuilderDevice::OptionKeyDuration,
-//     bpo::value<uint32_t>()->default_value(10000),
-//     "Time frame duration")
+    (o2::DataFlow::SubframeBuilderDevice::OptionKeyOrbitDuration,
+     bpo::value<uint32_t>()->default_value(o2::DataFlow::SubframeBuilderDevice::DefaultOrbitDuration),
+     "Orbit duration")
+    (o2::DataFlow::SubframeBuilderDevice::OptionKeyOrbitsPerTimeframe,
+     bpo::value<uint32_t>()->default_value(o2::DataFlow::SubframeBuilderDevice::DefaultOrbitsPerTimeframe),
+     "Orbits per timeframe")
     (o2::DataFlow::SubframeBuilderDevice::OptionKeySelfTriggered,
      bpo::bool_switch()->default_value(false),
-     "Time frame duration")
+     "Test timeframe")
```

Comment:
lets get rid of this option, this was a simple mockup in the beginning of the development